### PR TITLE
Replaces Collect with Checkout

### PR DESCRIPTION
**Pull request by dosdebug**

### CODE REVIEW

File: docs/big-picture.md [hunk 1]
```diff
@@ -8,8 +8,8 @@ was the **close-ended book holding** process:
 Let's briefly walk through it:
 - A **Regular Patron** can **place a book on a close-ended hold**  
 - A **Regular Patron** might **reach a maximum holds number** after a **hold is placed**  
-- A **Regular Patron** can either **cancel the hold** or **collect the book**  
-- While the book is **collected** the **hold is completed** and so the **returning process** starts  
+- A **Regular Patron** can either **cancel the hold** or **check out the book**  
+- While the book is **checkedout** the **hold is completed** and so the **returning process** starts
```
Comment (bslota) [inline docs/big-picture.md]: space missing: `checked out`

File: docs/big-picture.md [hunk 2]
```diff
@@ -23,8 +23,8 @@ Similar discoveries were made around **open-ened book holding** process:
 ![Open ended book holding](images/es/bigpicture/open-ended-holding-process.png)  
 
 - A **Researcher Patron** can **place a book on an open-ended hold**
-- A **Researcher Patron** can either **cancel the hold** or **collect a book**
-- While the book is **collected** the **hold is completed** and so the **returning process** starts
+- A **Researcher Patron** can either **cancel the hold** or **checkout a book**
+- While the book is **checkedout** the **hold is completed** and so the **returning process** starts
```
Comment (bslota) [inline docs/big-picture.md]: `checked out`

File: src/integration-test/groovy/io/pillopl/library/lending/dailysheet/infrastructure/FindingHoldsInDailySheetDatabaseIT.groovy [hunk 3]
```diff
@@ -97,13 +97,13 @@ class FindingHoldsInDailySheetDatabaseIT extends Specification {
             readModel.queryForHoldsToExpireSheet().count() == currentNoOfExpiredHolds
     }
 
-    def 'should never find already collected holds'() {
+    def 'should never find already checkedOut holds'() {
```
Comment (bslota) [inline src/integration-test/groovy/io/pillopl/library/lending/dailysheet/infrastructure/FindingHoldsInDailySheetDatabaseIT.groovy]: `checked out`

**Comment by dosdebug** [comment]
@bslota Done.

**Comment by bslota** [comment]
@dosdebug Excellent job! Thank you!

**Pull request merged**